Public/Sdk/SelfHost/RuntimeContracts/RuntimeContracts.dsc

```diff
@@ -40,7 +40,7 @@ export function withRuntimeContracts(args: Managed.Arguments, contractsLevel?: C
     return args.merge<Managed.Arguments>({
         defineConstants: getContractsSymbols(contractsLevel || ContractLevel.full, isDebug),
         references: [
-            importFrom("RuntimeContracts").pkg
+            importFrom("RuntimeContracts").withQualifier({targetFramework: 'netstandard2.0'}).pkg

```

Comment:
>").withQualifier({targetFramewor [](start = 40, length = 32)

same question... why force netstandard?

Reply:
Same answer as above, package has `.NETStandard` assemblies only, hence wouldn't coerce to full framework qualifiers and we explicitly choose what to use.